<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryWidget</class>
 <widget class="QWidget" name="HistoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="doneLabel">
       <property name="text">
        <string>Total sheets done:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="doneCount">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="avgLabel">
       <property name="text">
        <string>Average per day:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="avgCount">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Group by job</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Group by day</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="refreshButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
     <property name="icon">
      <iconset theme="view-refresh">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTreeView" name="treeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>treeView</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>refreshButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>refreshButton</sender>
   <signal>clicked()</signal>
   <receiver>HistoryWidget</receiver>
   <slot>refreshClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>409</x>
     <y>540</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshClicked()</slot>
 </slots>
</ui>
